paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/cdc/mysql/MySqlDebeziumJsonEventParser.java

```diff
@@ -67,8 +68,8 @@ public class MySqlDebeziumJsonEventParser implements EventParser<String> {
     private final List<ComputedColumn> computedColumns;
 
     private JsonNode payload;
-    private Map<String, String> mySqlFieldTypes;
-    private Map<String, String> fieldClassNames;
+    private Map<String, Map<String, String>> mySqlFieldTypesMap = Maps.newConcurrentMap();
+    private Map<String, Map<String, String>> fieldClassNamesMap = Maps.newConcurrentMap();
```

Comment:
Add a comment like `table name -> (field name -> type)` to make it easier to understand?